src/test/scala/mesosphere/marathon/state/AppDefinitionPortAssignmentsTest.scala

```diff
@@ -84,17 +84,6 @@ class AppDefinitionPortAssignmentsTest extends UnitTest {
       task.status.networkInfo.portAssignments(app, includeUnresolved = true) should be(empty)
     }
 
-    "portAssignments with a reserved task" in {
```

Comment:
Where did this go?

Reply:
reserved instances don't have tasks anymore...

Reply:
or at least does not have to have...